src/hyppo/dos.asm

```diff
@@ -835,6 +835,8 @@ trap_dos_openfile:
 
         +Checkpoint "trap_dos_openfile <success>"
 
+        lda dos_current_file_descriptor
+        sta hypervisor_a
```

Comment:
This pattern is showing up in a few places now, we could save bytes by having a return_from_trap_current_file_descriptor_and_success or similar.